modules/ossm-federation-overview.adoc

```diff
@@ -0,0 +1,17 @@
+////
+This module included in the following assemblies:
+- ossm-federation.adoc
+////
+
+
+[id="ossm-federation-overview_{context}"]
+= Federation overview
+
+[role="_abstract"]
+Federation is a set of features that let you connect services between separate meshes, allowing the use of {ProductShortName} features such as authentication, authorization, and traffic management across multiple, distinct administrative domains.
+
+Implementing a federated mesh lets you run, manage, and observe a single service mesh running across multiple OpenShift clusters.  {ProductName} federation takes an opinionated approach to a multi-cluster implementation of Service Mesh that assumes _minimal_ trust between meshes.
```

Comment:
@JStickler what does "an opinionated approach" mean ? Do we have a more clear way to describe a basic approach ?

Reply:
An opinionated approach means that we have an opinion about how to configure the software (in this case, that it is safer to have minimal trust between meshes).

Reply:
"opinionated" is a term I see fairly widely in software dev communities to mean "we feel this is the best way...". I often prefer the term "guided" (who likes overly opinionated people?), but I think it works in this case, especially as it's explained in the next paragraph. If you wanted to be more precise, I suppose you could also say a "secure" approach or a "multi-tenant approach"... but I'm ok leaving it as is.

Reply:
s/_minimal_/minimal